From horms@verge.net.au  Thu Jan 14 16:16:04 2010
From: Simon Horman <horms@verge.net.au>
Date: Wed, 23 Dec 2009 19:54:53 +1100
Subject: Staging: rtl8192e: remove some functions from the __exit section
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20091223085553.089696530@vergenet.net>


ieee80211_crypto_tkip_exit(), ieee80211_crypto_deinit() and
ieee80211_crypto_ccmp_exit() are called by ieee80211_rtl_init()
which are in section __init, so they can't be in section __exit.

Signed-off-by: Simon Horman <horms@verge.net.au>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/rtl8192e/ieee80211/ieee80211_crypt.c      |    2 +-
 drivers/staging/rtl8192e/ieee80211/ieee80211_crypt_ccmp.c |    2 +-
 drivers/staging/rtl8192e/ieee80211/ieee80211_crypt_tkip.c |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/rtl8192e/ieee80211/ieee80211_crypt.c
+++ b/drivers/staging/rtl8192e/ieee80211/ieee80211_crypt.c
@@ -225,7 +225,7 @@ out:
 }
 
 
-void __exit ieee80211_crypto_deinit(void)
+void ieee80211_crypto_deinit(void)
 {
 	struct list_head *ptr, *n;
 
--- a/drivers/staging/rtl8192e/ieee80211/ieee80211_crypt_ccmp.c
+++ b/drivers/staging/rtl8192e/ieee80211/ieee80211_crypt_ccmp.c
@@ -524,7 +524,7 @@ int __init ieee80211_crypto_ccmp_init(vo
 }
 
 
-void __exit ieee80211_crypto_ccmp_exit(void)
+void ieee80211_crypto_ccmp_exit(void)
 {
 	ieee80211_unregister_crypto_ops(&ieee80211_crypt_ccmp);
 }
--- a/drivers/staging/rtl8192e/ieee80211/ieee80211_crypt_tkip.c
+++ b/drivers/staging/rtl8192e/ieee80211/ieee80211_crypt_tkip.c
@@ -1012,7 +1012,7 @@ int __init ieee80211_crypto_tkip_init(vo
 }
 
 
-void __exit ieee80211_crypto_tkip_exit(void)
+void ieee80211_crypto_tkip_exit(void)
 {
 	ieee80211_unregister_crypto_ops(&ieee80211_crypt_tkip);
 }
